recipes/astra-toolbox/bld.bat

```diff
@@ -0,0 +1,33 @@
+@echo off
+
+set R=%SRC_DIR%
+
+set B_VC=C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\
+rem call "%B_VC%\vcvars64.bat" amd64 8.1 -vcvars_ver=14.0
+
+set B_BV=1_69
+set B_BOOST=E:\wjp\boost_%B_BV%_0
```

Comment:
Why?

Reply:
I don't know, I was trying to accomplish the build with the least amount of work by copying the provided build scripts from the source repo. 😅